ui/src/notifications/endpoints/components/EndpointTypeDropdown.tsx

```diff
@@ -40,7 +48,10 @@ const EndpointTypeDropdown: FC<Props> = ({
   blockedEndpoints,
 }) => {
   const items = types
-    .filter(({type}) => !blockedEndpoints.includes(type))
+    .filter(
+      ({type, flag}) =>
+        !blockedEndpoints.includes(type) && (!flag || isFlagEnabled(flag))
```

Comment:
let's create another function that calls `isFlagEnabled`, that gets passed the type of the endpoint. 
```suggestion
        !blockedEndpoints.includes(type) && (isFlaggedOn(type))
```